replit_river/error_schema.py

```diff
@@ -21,3 +21,24 @@ def __init__(self, code: str, message: str) -> None:
         self.code = code
         self.message = message
         super().__init__(f"Error in river, code: {code}, message: {message}")
+
+
+def stringify_exception(e: BaseException, limit: int = 10) -> str:
+    """Return a string representation of an Exception.
+
+    This is different from just calling str(e) because it will also show the chained exceptions as
```

Comment:
huge

Reply:
might be nice to do first 5 last 5 causes if its more than 10?

Reply:
typically the topmost are kinda useless since they typically live deep inside the machinery of asyncio, lol

Reply:
(by "topmost" i mean Python-style because "most recent call last")